wordpress-mobile/WordPress-iOS
# Don’t allow to switch html mode when media is uploading.

Fixes #669

To test:
 - Start a post using the Aztec editor
 - Add a media object
 - While the upload is happening try to switch to html
 - See if an alert appears that blocks the switch.
 - Wait for upload to finish
 - Try to switch again.

Needs review: @jleandroperez
